source/stepbro/CommandLineOptions.cs

```diff
@@ -27,8 +27,11 @@ internal class CommandLineOptions : StepBro.Core.General.CoreCommandlineOptions
         [Option('w', "wait", Default = false, HelpText = "Awaits key press before terminating.")]
         public bool AwaitKeypress { get; set; } = false;
 
-        [Option("report_result", Default = false, HelpText = "Dumps the execution result.")]
-        public bool ReportResult { get; set; } = false;
+        //[Option("report_result", Default = false, HelpText = "Dumps the execution result.")]
+        //public bool ReportResult { get; set; } = false;
```

Comment:
If this is unused, should we just remove it, or do we plan to use it in the future?